docs/features/onboarding.md

```diff
@@ -0,0 +1,61 @@
+# WooCommerce Onboarding
+
+The onboarding feature is a reimagined new user setup experience for WooCommerce core. It contains two sections aimed at getting merchants started with their stores. The merchant begins with the "profile wizard", which helps with initial steps like setting a store address, making extension recommendations, and connecting to Jetpack for additional features. Once the profile wizard is complete, the merchant can purchase & install optional extensions via WooCommerce.com, before continuing to the "task list". From the task list, merchants are walked through a variety of items to help them start selling, such as adding their first product and setting up payment methods.
+
+* Master Thread: p6q8Tx-R1-p2
+* Designs: 2317a-pb
+
+## Enabling Onboarding
+
+If you run the development version of WooCommerce Admin from GitHub directly, no further action should be needed to enable Onboarding.
+
+Users of the published WooCommerce Admin plugin need to either opt-in to using the new onboarding experience manually, or become part of the a/b test in core. See https://github.com/woocommerce/woocommerce/pull/24991 for more details on the testing implementation.
+
+To enable the new onboarding experience manually, log-in to `wp-admin`, and go to `WooCommerce > Settings > Help > Setup Wizard`. Click `Enable` under the `New onboarding experience` heading.
+
+You can also set the following configuration flag in your `wp-config.php`:
+
+`define( 'WOOCOMMERCE_ADMIN_ONBOARDING_ENABLED', true );`
+
+## New REST API endpoints
+
+@todo
+
+## Onboarding filters and hooks
+
+@todo
+
+## Options and settings
+
+@todo
+
+## WCCOM connection and redirection
+
+During the profile wizard, merchants can select paid product type extensions (like WooCommerce Memberships) or a paid theme. To make installation easier and to finish purchasing, it is necessary to make a [WooCommerce.com connection](https://docs.woocommerce.com/document/managing-woocommerce-com-subscriptions/). We also prompt users to connect on the task list if they chose extensions in the profile wizard, but did not finish connecting.
+
+To make the connection from the new oboarding experience possible, we build our own connection endpoints [/wc-admin/v1/request-wccom-connect](https://github.com/woocommerce/woocommerce-admin/blob/61b771c2643c24334ea062ab3521073beaf50019/src/API/OnboardingPlugins.php#L298-L355) and [/wc-admin/v1/finish-wccom-connect](https://github.com/woocommerce/woocommerce-admin/blob/61b771c2643c24334ea062ab3521073beaf50019/src/API/OnboardingPlugins.php#L357-L417).
```

Comment:
The `v1/` was removed from the API namespace.

Reply:
Thanks for catching that!